packages/workbox-precaching/default-precaching-export.mjs

```diff
@@ -0,0 +1,30 @@
+import PrecacheController from './controllers/PrecacheController.mjs';
+
+const precacheController = new PrecacheController();
+let listenersAdded = false;
+
+const addListeners = () => {
+  listenersAdded = true;
+  self.addEventListener('install', () => precacheController.install());
```

Comment:
For both `install` and `activate`, you should either:
- pass in the corresponding `event` parameter to the `precacheController` method and call `event.waitUntil()` within that method
- or wrap the call to the `precacheController` methods in `event.waitUntil()` right here

Since both `precacheController.install()` and `.cleanup()` are `async` methods, it probably makes more sense to take the second approach:

```js
self.addEventListener('install', (event) => {
  event.waitUntil(precacheController.install());
});
```

Reply:
Such a noob mistake - thanks Jeff.